scrapers/JaysPov.yml

```diff
@@ -0,0 +1,30 @@
+name: "Jays POV"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - jayspov.net
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="container-fluid video-title"]/h1/text()
+        replace:
+          - regex: '^(.+?)\ \-\ (.+)$'
+            with: $2
+      Details: //meta[@name="og:description"]/@content
+      Date:
+        selector: //div[@class="release-date"]/text()
```

Comment:
The date selector gets an extra unwanted field.
Switching to
```yaml
selector: //div[@class="release-date" and contains(.,"Released:")]/text()
```
seems to work better